docs/stable/9-blockchain-usage.md

```diff
@@ -0,0 +1,184 @@
+---
+permalink: /spec/9
+parent: Stable specs
+title: 9/BLOCKCHAIN-USAGE
+---
+
+# 3/BLOCKCHAIN-USAGE
+
+> Version: 0.1
+>
+> Status: Stable
+>
+> Authors: Andrea Maria Piana <andreap@status.im>
+
+# Status interactions with the blockchain
+
+In this document is document all the interactions that the Status client has
+with the blockchain.
+
+1. [Wallet](#Wallet)
+2. [ENS](#ENS)
+
+## Wallet
+
+The wallet in Status has two main components:
+
+1) Sending transactions
+2) Fetching balance
```

Comment:
Signing messages would be another function

Reply:
Signing messages as signing whisper messages (not a function of wallet)? or signing messages as to sign some custom data? (which we use to authenticate transactions addresses), 
NeitherI believe interact with the blockchain,
I can change the sentence to:
`The wallet in Status has two main components that interacts with the blockchain:` if  that's the case

Reply:
meta transactions

Reply:
do we support meta transactions?